logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/featureConfig/SyncFeatureConfigsUseCase.kt

```diff
@@ -131,14 +135,24 @@ internal class SyncFeatureConfigsUseCaseImpl(
 
     private fun handleSelfDeletingMessagesStatus(model: SelfDeletingMessagesModel) {
         if (!kaliumConfigs.selfDeletingMessages) {
-            userConfigRepository.setSelfDeletingMessagesStatus(SelfDeletingMessagesStatus(false, null, null))
+            userConfigRepository.setTeamSettingsSelfDeletionStatus(
+                TeamSettingsSelfDeletionStatus(
+                    enforcedSelfDeletionTimer = SelfDeletionTimer.Disabled,
+                    hasFeatureChanged = null
```

Comment:
I honestly still think that variable name is abit confusing, perhaps it is me lacking context, but just looking at this I have no idea how this works, what means hasFeatureChanged ? Since the last time we create UserSessionScope ? Since the last time user launched the app ? Maybe a better name would give more context of what this purpose is, perhaps : hasDeletionTimerChangedSinceLastLaunch ? or something, not sure how to tackle this. 

Kind of minor thing, but I can see it becoming place for unclearity

Reply:
This is a variable used to identify whether the team settings have been changed recently in order to show an informative dialog on the Home Screen, in a similar fashion than other feature flags do it. This will be set again to `false` once the user has seen the dialog and acknowledges it by dismissing it. I added a documentation line on the object to be more redundant in case of new future confusions.